src/arrayMethodSort.js

```diff
@@ -3,9 +3,39 @@
 /**
  * Implement method Sort
  */
+const compareAsString = (a, b) => {
+  const stringA = String(a);
+  const stringB = String(b);
+
+  if (stringA > stringB) {
+    return 1;
+  } else if (stringA === stringB) {
+    return 0;
+  } else {
+    return -1;
+  }
```

Comment:
Do not use else statement
```suggestion
  if (stringA > stringB) {
    return 1;
  }

  if (stringA < stringB){
    return -1;
  }

  return 0;
```